### PR TITLE
CLIENT PAUSE and  type argument in client_list

### PR DESCRIPTION
**Pull request by RoeyPrat**

### CODE REVIEW

File: redis/client.py [hunk 1]
```diff
@@ -785,8 +786,20 @@ def client_kill(self, address):
...
+    def client_list(self, _type=None):
+        """
+        Returns a list of currently connected clients.
+        If type of client specified, only that type will be returned.
+        :param _type: optional. one of the client types (normal, master,
+         replica, pubsub)
+        """
         "Returns a list of currently connected clients"
+        if _type is not None:
+            if _type not in ['normal', 'master', 'replica', 'pubsub']:
```
Comment (itamarhaber) [inline redis/client.py]: Perhaps `if str(_type).lower() not in ['normal', 'master', 'replica', 'pubsub']:`?

Comment (andymccurdy) [reply redis/client.py]: Agreed with Itamar.

Comment (itamarhaber) [reply redis/client.py]: Resolved via 0adcc53

**Comment by itamarhaber** [comment]
@andymccurdy you have a clear path to happiness.

**Pull request merged**